bazel/esbuild/esbuild-amd-config.mjs

```diff
@@ -0,0 +1,24 @@
+import url from 'url';
+import path from 'path';
+
+/** Path to the ESBuild configuration maintained by the user. */
+const userConfigExecPath = 'TMPL_CONFIG_PATH';
+
+/** User ESBuild config. Empty if none is loaded. */
+let userConfig = {};
+
+if (userConfigExecPath !== '') {
+  const userConfigPath = path.join(process.cwd(), userConfigExecPath);
+  const userConfigUrl = url.pathToFileURL(userConfigPath);
+
+  // Load the user config, assuming it is set as `default` export.
```

Comment:
Can we have this comment note that it is expecting the `default` export because we define it in our own file.